YOUR_ADMIN_FOLDER/widget.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+* @package tawk.to
+* @copyright Copyright 2021 tawk.to
+* @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
+* @version 1.0.1
+*/
+
+require('includes/application_top.php');
+
+if (isset($_GET['set'])) {
```

Comment:
Instead of using `isset($_GET['set'])`, providing `cmd` or action param and then selecting from 2 functions to be executed is the way to go here IMHO